docs/modules/ROOT/pages/getting-spring-security.adoc

```diff
@@ -15,19 +15,22 @@ Typically, these are done to provide improved security to match modern security
 
 
 [[maven]]
-== Usage with Maven
+== Usage with Maven and Gradle
 
-As most open source projects, Spring Security deploys its dependencies as Maven artifacts.
-The topics in this section describe how to consume Spring Security when using Maven.
+As most open source projects, Spring Security deploys its dependencies as Maven artifacts, which makes them compatible with both Maven and Gradle. The following sections demonstrate how to integrate Spring Security with these build tools, with examples for Spring Boot and standalone usage.
 
 [[getting-maven-boot]]
-=== Spring Boot with Maven
+[[getting-gradle-boot]]
+=== Spring Boot with Maven and Gradle
```

Comment:
Since you've already stated that you'll be talking about both, it's okay to simply say `Spring Boot`.